<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPage</class>
 <widget class="QWidget" name="UserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2c3e50;
    color: #ecf0f1;
}

QLabel {
    color: #ecf0f1;
    font-size: 14px;
    font-weight: bold;
    background-color: transparent;
}

QLineEdit {
    padding: 12px;
    border-radius: 6px;
    background-color: rgba(52, 73, 94, 0.8);
    border: 2px solid #3498db;
    color: #ecf0f1;
    font-size: 14px;
    selection-background-color: #3498db;
}

QLineEdit:focus {
    border: 2px solid #2980b9;
    background-color: rgba(44, 62, 80, 0.9);
}

QLineEdit::placeholder {
    color: #95a5a6;
}

QPushButton {
    padding: 12px;
    border-radius: 6px;
    color: white;
    font-weight: bold;
    font-size: 14px;
    border: none;
    min-width: 120px;
}

QPushButton#loginButton {
    background-color: #2ecc71;
}

QPushButton#loginButton:hover {
    background-color: #27ae60;
}

QPushButton#loginButton:pressed {
    background-color: #219a52;
}

QPushButton#registerButton {
    background-color: #3498db;
}

QPushButton#registerButton:hover {
    background-color: #2980b9;
}

QPushButton#registerButton:pressed {
    background-color: #2472a4;
}

#titleLabel {
    color: #3498db;
    font-size: 24px;
    font-weight: bold;
    margin: 20px;
    background-color: transparent;
}

#formContainer {
    background-color: rgba(52, 73, 94, 0.7);
    border-radius: 10px;
    padding: 20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>碳足迹计算器 - 用户中心</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="formContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="usernameLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="usernameInput">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>请输入用户名</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="passwordInput">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>请输入密码</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="emailLabel">
          <property name="text">
           <string>邮箱</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="emailInput">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>请输入邮箱（仅注册需要）</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registerButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #e74c3c;

QPushButton#logoutButton:hover {
    background-color: #c0392b;
}

QPushButton#logoutButton:pressed {
    background-color: #a93226;
}</string>
        </property>
        <property name="text">
         <string>退出登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 